source/clear-linux/tutorials/multi-boot/multi-boot-sles.rst

```diff
@@ -0,0 +1,128 @@
+.. _multi-boot-sles:
+
+Install SUSE\* Linux Enterprise 12 SP2
+**************************************
+
+#. Start the SUSE installer and follow the prompts.
+
+#. At the :guilabel:`Suggested Partitioning` screen, choose
+   :guilabel:`Expert Partitioner`. See Figure 1.
+
+   .. figure:: figures/multi-boot-sles-1.png
+
+      Figure 1: SUSE: Suggested partitioning
+
+   **Optional:** Under :guilabel:`Available Storage on Linux` section,
+   right-click the SUSE :file:`/home` partition and delete it. In this example, it is :file:`/dev/sda8`. See Figure 2.
+
+   .. figure:: figures/multi-boot-sles-2.png
+
+      Figure 2: SUSE: Delete /home partition
+
+#. Under :guilabel:`Available Storage on Linux` section, right-click the SUSE
+   root partition and resize it. In this example, :file:`/dev/sda7` is
+   resized to 45 GB. See Figure 3.
+
+   .. figure:: figures/multi-boot-sles-3.png
+
+      Figure 3: SUSE: Resize root partition
+
+#. Click :guilabel:`Accept`.
+
+#. Follow the remaining prompts to complete the installation of SUSE.
+
+#. At this point, |CL| cannot boot because `Grub`
+   is the default boot loader. Follow these steps to make the |CL|
+   Systemd-Boot the default boot loader and add SUSE as a boot option:
+
+   #. Boot into SUSE.
+
+   #. Log in.
+
+   #. Get root privileges with the following command:
+
+      .. code-block:: console
+
+         $ sudo -s
+
+   #. Locate SUSE’s :file:`grub.cfg` in the :file:`/boot/grub2/` directory
+      and look for the primary SUSE :guilabel:`menuentry` section. The
+      highlighted lines identify the kernel, the :file:`initrd` filenames,
+      the root partition UUID, and the additional parameters used. Use this
+      information to create a new Systemd-Boot entry. See Figure 4.
+
+      .. figure:: figures/multi-boot-sles-4.png
+
+         Figure 4: SUSE: grub.cfg
+
+   #. Copy the kernel and the :file:`initrd` file to the EFI partition.
+
+      .. code-block:: console
+
+         # cp /boot/vmlinuz-4.4.21-69-default /boot/efi
```

Comment:
Change "# cp /boot/vmlinuz-4.4.21-69-default /boot/efi" to "$ sudo cp /boot/vmlinuz-4.4.21-69-default /boot/efi"

Reply:
Text edits deferred to next phase of updates.